lbr_fri_ros2/include/lbr_fri_ros2/filters.hpp

```diff
@@ -126,20 +126,18 @@ class JointPIDArray {
   using pid_array_t = std::array<control_toolbox::Pid, KUKA::FRI::LBRState::NUMBER_OF_JOINTS>;
 
 public:
-  JointPIDArray() = default;
+  JointPIDArray() = delete;
```

Comment:
could you please make sure to pull the latest changes from the `humble` branch? 

So that no changes are made to the driver.

Reply:
Thanks @mhubii. I'm a little confused. Didn't you change the pid parameters and how they are configured based on [the Issue](https://github.com/lbr-stack/lbr_fri_ros2_stack/issues/187#issuecomment-2160115765) in [this commit](https://github.com/lbr-stack/lbr_fri_ros2_stack/commit/f4943ebb78f2c1d1f804a8b7345f63e59127e274)? 
I was checking [my lbr_system_parameters.yaml](https://github.com/OmidRezayof/lbr_fri_ros2_stack/blob/cartesian_vel/lbr_ros2_control/config/lbr_system_parameters.yaml) with [yours](https://github.com/lbr-stack/lbr_fri_ros2_stack/blob/humble/lbr_ros2_control/config/lbr_system_parameters.yaml) and they're the same?